src/pages/index.js

```diff
@@ -1,15 +1,17 @@
 import "./index.css";
 /* ------------------------- Import all the classes ------------------------- */
+import Api from "../components/Api.js";
 import { initialCards, selectors } from "../utils/constants.js";
 import FormValidator from "../components/FormValidator.js";
 import Card from "../components/Card.js";
 import Section from "../components/Section.js";
-import Popup from "../components/Popup.js";
+// import Popup from "../components/Popup.js";
 import PopupWithImage from "../components/PopupwithImage.js";
 import PopupWithForm from "../components/PopupWithForm.js";
 import UserInfo from "../components/UserInfo.js";
 import { validationSettings } from "../utils/constants.js";
-/* ------------------------- Set up all the classes ------------------------- */
+import PopupWithConfirm from "../components/PopupWithConfirm.js";
+// -------------- Set up all the Validators & Constants ------------------------- */
 
 const profileEditModal = document.querySelector("#profile-edit-modal");
 const profileEditForm = profileEditModal.querySelector(".modal__form");
```

Comment:
**COULD BE IMPROVED**

We can get any form using their **unique** attribute `name` or `id` from `document.forms`

```js

const profileForm = document.forms["profile-form"];
const cardForm = document.forms["card-form"];

```

You need to add `name` or `id` first in `index.html` to achieve that.

More info here https://developer.mozilla.org/en-US/docs/Web/API/Document/forms